src/main/java/frc/team88/swerve/gyro/Pigeon.java

```diff
@@ -16,6 +16,15 @@ public Pigeon() {
     this.base = new PigeonIMU(0);
   }
 
+  /**
+   * Constructor. Uses the given PigeonIMU object instead of constructing a new one.
+   *
+   * @param base The AHRS object to base this object on.
```

Comment:
Should probably be Pigeon rather than AHRS in the parameter

Reply:
```suggestion
   * @param base The PigeonIMU object to base this object on.
```